### PR TITLE
CNV-12361: Adding Quick Start section to New Features in RN

### PR DESCRIPTION
**Pull request by bgaydosrh**
for 4.8 only.

Jira: https://issues.redhat.com/browse/CNV-12361

Code and QE reviews passed. See original PR: https://github.com/openshift/openshift-docs/pull/34299 

Direct doc preview link: https://deploy-preview-34611--osdocs.netlify.app/openshift-enterprise/latest/virt/virt-4-8-release-notes.html#virt-4-8-quick-starts

### CODE REVIEW

File: virt/virt-4-8-release-notes.adoc [hunk 1]
```diff
@@ -64,6 +64,12 @@ The SVVP Certification applies to:
 
 * {VirtProductName} now supports third-party Container Network Interface (CNI) plug-ins that are link:https://access.redhat.com/articles/5436171[certified by Red Hat] for use with {product-title}.
 
+[id="virt-4-8-quick-starts"]
+=== Quick starts
+
+* Quick start tours are available for several {VirtProductName} features. To view the tours, click the *?* icon in the masthead of the {VirtProductName} console and select *Quick Starts*.
```
Comment (sjhala-ccs) [inline virt/virt-4-8-release-notes.adoc:70]: Does the **?** icon have a generic name to go with its function or does it use a tooltip? Per ISG, the graphic for the UI element should come after the label or tooltip name. For example, you could say, click the **Help** icon **?**.

Comment (sjhala-ccs) [inline virt/virt-4-8-release-notes.adoc:70]: I wasn't familiar with the term "masthead". Do you think our customers would know where to look? Would it make sense to simplify the location to say "menu bar on the header"?

**Review by sjhala-ccs — COMMENTED** [review]
Just one nit and a suggestion for you to take or leave. Thanks, Bob!

File: virt/virt-4-8-release-notes.adoc (hunk 1 above)
Comment (bgaydosrh) [reply virt/virt-4-8-release-notes.adoc:70]: great, thanks.

Comment (bgaydosrh) [reply virt/virt-4-8-release-notes.adoc:70]: Done

**Pull request merged**